### PR TITLE
Fix upgrade paths for 11.0

### PR DESCRIPTION
**Pull request by agedemenli**
Fixing the upgrade path for version 11.0

### CODE REVIEW

File: src/backend/distributed/sql/downgrades/citus--10.2-5--10.2-4.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "../udfs/citus_finish_pg_upgrade/10.2-4.sql"
```
Comment (hanefi) [inline src/backend/distributed/sql/downgrades/citus--10.2-5--10.2-4.sql:1]: We already have the same file in the parent directory. I think we should remove this one.

File: src/test/regress/sql/multi_extension.sql [hunk 2]
```diff
@@ -395,6 +395,14 @@ SELECT * FROM multi_extension.print_extension_changes();
...
 SELECT * FROM multi_extension.print_extension_changes();
 
+-- Snapshot of state at 10.2-5
+ALTER EXTENSION citus UPDATE TO '10.2-5';
+SELECT * FROM multi_extension.print_extension_changes();
+
+-- Test downgrade to 10.2-4 from 10.2-5
+ALTER EXTENSION citus UPDATE TO '10.2-4';
+ALTER EXTENSION citus UPDATE TO '10.2-5';
+
```
Comment (hanefi) [inline src/test/regress/sql/multi_extension.sql:405]: I like these lines. However some of the next tests also need to be updated now. Please go over the occurrences of `10.2-4` later in the file and change them into `10.2-5` if it makes sense to you.

Some examples:

> -- Test downgrade to 10.2-4 from 11.0-1

> ALTER EXTENSION citus UPDATE TO '10.2-4';

**Review by hanefi — CHANGES_REQUESTED** [review]

**Comment by hanefi** [comment]
One final comment, and we are good to go.

You renamed a file, but the comment in it became stale now. The comment does not match the file name.

https://github.com/citusdata/citus/blob/8bb0dee7f361d15fc884842551305fca7debf213/src/backend/distributed/sql/citus--10.2-5--11.0-1.sql#L1

I think you can continue working on the other branches after this fix is over. Let's try to merge 3 PRs after all of them are ready.

**Comment by agedemenli** [comment]
> One final comment, and we are good to go.
> 
> You renamed a file, but the comment in it became stale now. The comment does not match the file name.
> 
> https://github.com/citusdata/citus/blob/8bb0dee7f361d15fc884842551305fca7debf213/src/backend/distributed/sql/citus--10.2-5--11.0-1.sql#L1
> 
> I think you can continue working on the other branches after this fix is over. Let's try to merge 3 PRs after all of them are ready.

Missed that, thank you!
Fixed now.
Will let you know when the other ones are ready as well

**Review by hanefi — APPROVED** [review]
Let's wait until the changes for `main` and `release-10.2` are ready.

We better squash the commits, and have a descriptive commit message.

**Pull request merged**